jenkins/PerfCI_Operators_Deployment/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+pipeline {
+    agent {
+        label 'PerfCi'
+    }
+    environment {
+        CONTACT_EMAIL1 = credentials('perfci_contact_email1')
+        PROVISION_USER = credentials('perfci_provision_user')
+        PROVISION_IP = credentials('perfci_provision_ip')
+        WORKER_DISK_IDS = credentials('perfci_worker_disk_ids')
+        WORKER_DISK_PREFIX = credentials('perfci_worker_disk_ids_prefix')
+        KUBEADMIN_PASSWORD = readFile('/home/jenkins/.kube/kubeadmin-password').trim()
+        KUBEADMIN_PASSWORD_PATH = '/home/jenkins/.kube/kubeadmin-password'
+        KUBECONFIG_PATH = '/root/.kube/config'
+        PRIVATE_KEY_PATH = '/home/jenkins/.ssh/provision_private_key'
+        CONFIG_PATH = '/home/jenkins/.ssh/config'
+        WORKSPACE='/home/jenkins'
+        CONTAINER_PRIVATE_KEY_PATH = '/root/.ssh/provision_private_key'
+        PROVISION_PORT = 22
+        CNV_VERSION = '4.14'
+        ODF_VERSION = '4.13'
```

Comment:
These should go into the environment variable or secrets list; versions should not be scattered about the code.

Reply:
Thats the way we are doing it also in IBM Clouds, the versions are visible:
https://github.com/redhat-performance/benchmark-runner/blob/5781a5d67a14c3de0030ff923709e47f584d0148/.github/workflows/Weekly_Perf_Env_Operator_CI.yml#L80-L81

I believe it's unnecessary to keep the version a secret; knowing which version we have should not be a concern.

Reply:
If we put it in the secret we would know the current installed version, Are u sure its good to make it as secret ?

Reply:
Secret, environment variable, whatever.  Just not in code.

Reply:
Change it, it means that no need to open PR for upgrading new version anymore.
We need only update the secret, is that ok ?